fighters/common/src/function_hooks/ledges.rs

```diff
@@ -24,7 +24,7 @@ unsafe fn can_entry_cliff_hook(boma: &mut BattleObjectModuleAccessor) -> u64 {
     let status_kind = StatusModule::status_kind(boma);
     let fighter_kind = boma.kind();
 
-    let rising: f32 = KineticModule::get_sum_speed_y(boma, *FIGHTER_KINETIC_ENERGY_ID_GRAVITY); // Rising while jumping/airdodging
+    let rising: f32 = KineticModule::get_sum_speed_y(boma, *KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN); // Rising while jumping/airdodging

```

Comment:
curious what effects this change has

Reply:
No change, it derefs to the same integer value as before lmao, whoever wrote the initial code mistakenly used that const, even tho it worked…
get_sum_speed funcs take ENERGY_RESERVE_ATTRIBUTE consts as an arg